src/molecules/side-menu.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from './side-menu.module.css';
+
+const SideMenu = ({ isOpen, closeMenu, subjects }) => {
+  return (
+    <>
+      {isOpen && <div className={`${styles.overlay} scrim`} onClick={closeMenu}></div>}
+      <div className={`${styles.sideMenu} surface`} data-open={isOpen}>
+        <div className={`${styles.topBar} on-surface`}>
```

Comment:
```suggestion
        <div className={`${styles.topBar} on-surface-text`}>
```
`-text` を付けないと `on-surface` の色が背景色に適用されてしまいます.